homeassistant/components/ihc/__init__.py

```diff
@@ -296,3 +314,6 @@ def set_runtime_value_float(call):
     hass.services.register(DOMAIN, SERVICE_SET_RUNTIME_VALUE_FLOAT,
                            set_runtime_value_float,
                            schema=SET_RUNTIME_VALUE_FLOAT_SCHEMA)
+    hass.services.async_register(DOMAIN, SERVICE_PULSE,
```

Comment:
Since we're in sync context, this should still be done using our sync api. Ie use `hass.services.register`.

Reply:
Fixed